### PR TITLE
Payment schema

### PR DESCRIPTION
**Pull request by ajkatom**
### What this PR does (required):
created payment Schema

### CODE REVIEW

File: server/utils/stripe.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+exports.createProduct = async () => {
+  const PRODUCT_NAME = 'Monthly Subscription';
+  const PRODUCT_TYPE = 'service';
+
+  const product = await stripe.products.create({
+    name: PRODUCT_NAME,
+    type: PRODUCT_TYPE,
+  });
+
+  console.log(product);
```
Comment (ezeilo-su) [inline server/utils/stripe.js:41]: Let's clean up our code and remove this console log.

File: server/utils/stripe.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  const plan = await stripe.plans.create({
+    product: productId,
+    nickname: PLAN_NICKNAME,
+    currency: CURRENCY,
+    interval: PLAN_INTERVAL,
+    amount: PLAN_PRICE,
+  });
+
+  console.log(plan);
```
Comment (ezeilo-su) [inline server/utils/stripe.js:58]: Remove this log too

File: server/utils/stripe.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  return plan.id;
+};
+
+exports.subscribeCustomerToPlan = async (customerId, planId) => {
+  const subscription = await stripe.subscriptions.create({
+    customer: customerId,
+    items: [{ plan: planId }],
+  });
+
+  console.log(subscription);
```
Comment (ezeilo-su) [inline server/utils/stripe.js:68]: Please remove

**Review by ezeilo-su — COMMENTED** [review]
Good work with this one

**Review by moffatethan — APPROVED** [review]
Please make sure to base the branches properly, the base branch for this PR should have been the `stripe` branch

**Review by moffatethan — CHANGES_REQUESTED** [review]
Please fix merge conflict 🙂

**Pull request merged**